### ISSUE TITLE
Editor content update

### ISSUE THREAD

**Issue by klauswr** [sinnlabs/zk-codemirror#3]
Hi there, you did a really great job! 
I am currently migrating from the other zk-codemirror implementation ([#1:](https://github.com/jease/zk-codemirror) to your implementation (#2) as it supports more recent codemirror versions.

Unfortunately there is ONE issue which might stop me from doing so: other than in the implementation #1 there is no way to refresh the content of the editor from the server: 

Example:
zul:
```
...
<codemirror id="editor" mode="@load(vm.mode)" value="@load(vm.content)" />
...
<button label="properties" onClick="@command('properties')" />
...
```
java:  
```
       ...
	@Command("properties")
	@NotifyChange({ "mode", "content" })
	public void onProperties() {
		setContent(_properties);
		setMode(CodeMirror.PROPERTIES);
	}
```

I tried also: 

```
	@Wire("#editor")
	private CodeMirror _editor;

	@AfterCompose
	public void afterCompose(@ContextParam(ContextType.VIEW) final Component view) {
		Selectors.wireComponents(view, this, false);
	}


...
	@Command("properties")
	@NotifyChange({ "mode", "content" })
	public void onProperties() {
		setContent(_properties);
		_editor.setValue(getContent());
		setMode(CodeMirror.PROPERTIES);
	}
...
```

Any chance to get this feature happen?

**Comment by nsa-a1** [comment]
Hi klauswr, yep i know about this bug. Code is already written. I need some time to test the release.

**Comment by nsa-a1** [comment]
Fixed in 0.2 release. Release uploaded to the Maven repo.

**Comment by klauswr** [comment]
Well done - works like a charm!

Thanks a lot.
